x/farming/keeper/staking.go

```diff
@@ -235,6 +243,25 @@ func (k Keeper) ReleaseStakingCoins(ctx sdk.Context, farmerAcc sdk.AccAddress, u
 	return nil
 }
 
+// afterStakingCoinAdded is called after a new staking coin denom appeared
+// during ProcessQueuedCoins.
+func (k Keeper) afterStakingCoinAdded(ctx sdk.Context, stakingCoinDenom string) error {
+	k.SetHistoricalRewards(ctx, stakingCoinDenom, 0, types.HistoricalRewards{CumulativeUnitRewards: sdk.DecCoins{}})
+	k.SetCurrentEpoch(ctx, stakingCoinDenom, 1)
+	k.SetOutstandingRewards(ctx, stakingCoinDenom, types.OutstandingRewards{Rewards: sdk.DecCoins{}})
+	return nil
+}
+
+// afterStakingCoinRemoved is called after a staking coin denom got removed
+// during Unstake.
+func (k Keeper) afterStakingCoinRemoved(ctx sdk.Context, stakingCoinDenom string) error {
+	// TODO: send remaining outstanding rewards to the fee pool
```

Comment:
In what situations do the outstanding rewards remain?
Rather than leaving it as `TODO:` why don't we create a separate issue or solve it in the PR?

Reply:
Actually I forgot to add that logic. I meant to do that in this PR, thank you for finding it out!
`OutstandingRewards` holds `sdk.DecCoins`, and there might be a little amount of coins left in that struct due to the decimal truncation, even if all farmers have withdrawn their rewards.
In `x/distribution`, it sends remaining (very little amount of)coins to the community pool, and in our case we could send it to the farming fee pool.